google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleDeploymentEditor.java

```diff
@@ -158,6 +158,9 @@ protected void textChanged(DocumentEvent event) {
     appYamlOverrideCheckBox.addActionListener(event -> {
       boolean isAppYamlOverrideSelected = ((JCheckBox) event.getSource()).isSelected();
       appYamlTextField.setVisible(isAppYamlOverrideSelected);
+
+      setModuleComboBoxAndButtonEnabled(
```

Comment:
Why do you suggest?
The method only sets the module bits to enabled/disabled, and I was basing on other methods like setEnabled().
I can replace Enabled by State?